### DISCUSSION TITLE
TAMAÑO DE LA PAGUINA CSS

### DISCUSSION THREAD

**Discussion by skrillheaven** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#429] [Q&A]
![duda](https://user-images.githubusercontent.com/74394439/158714190-4b8f595a-0b40-43a4-a2be-d431e8d1746c.png)

Buenas noches explorers, voy atrasado pero echándole ganas!!!, agradeceria su ayuda, alguien sabe como poder hacer que la paguina se vea uniforme, me sobra espacio en el lado derecho y quiero que el borde sea hasta la imagen, ya intente dandole tamaño al body, al html y al scale, gracias por la ayuda, son los mejores!!1

**Comment by AngelAlfaro92** [comment]
En tu Documento de CSS intenta poner lo siguiente pero al principio de todo
*{
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

**Reply by skrillheaven** [reply]
gracias por el aporte @AngelAlfaro92 , pero no se inmuto la pagina :(, de todas maneras gracias por tu valioso tiempo

**Comment by LuisEsquivel27** [comment]
Si con lo que comenta el compañero Angel, no se soluciona, podrías compartir capturas de tu html y tu css para que podamos ver que podría estar afectando.

**Reply by skrillheaven** [reply]
claro que siii, gracias de antemano, requieres todo el codigo?

**Reply by LuisEsquivel27** [reply]
no, solo el inicio, de hecho, como recomendación, para que no batalles en revisar todo el código, respalda lo que llevas, tu problema afecta a toda la pantalla, así que solo deja tu menú que tienes hasta arriba, si el problema sigue, pues ya es menos código que revisar y al solucionarlo ya puedes regresar todo tu avance.

**Reply by skrillheaven** [reply]
adjunto las capturas que considere importantes en el codigo
HTML
![html1](https://user-images.githubusercontent.com/74394439/158717094-b7d164a3-040e-49c3-9ebf-32589d58c51f.png)
![html2](https://user-images.githubusercontent.com/74394439/158717096-d4c1999a-0284-48ec-b90a-36986241bf65.png)
![html3](https://user-images.githubusercontent.com/74394439/158717098-9c500169-37ef-43d1-926f-78b42c6ea84a.png)
![html4](https://user-images.githubusercontent.com/74394439/158717099-ef4c13db-f60f-4498-9d87-6c03aafb93f5.png)
![html5](https://user-images.githubusercontent.com/74394439/158717100-9cbb9d9d-72cc-46ef-ae72-c82a01bb2b9d.png)

css

![css1](https://user-images.githubusercontent.com/74394439/158717142-9bfb55ce-7995-470d-8a99-ba77886a4ca4.png)
![css2](https://user-images.githubusercontent.com/74394439/158717145-f200d8d2-2731-4ed7-82e8-d6071ef3f5f0.png)
![css3](https://user-images.githubusercontent.com/74394439/158717147-0199ef2f-d414-4963-bb76-0c95914157ee.png)

**Reply by LuisEsquivel27** [reply]
![Captura](https://user-images.githubusercontent.com/26752004/158717460-28e14036-4e55-4354-8516-c58ec408ca57.PNG)
este es el detalle, al body le estas indicando que debe tener un width de 1000px, por eso se limita, solo quita body y deja tu clase .barra y prueba

**Reply by skrillheaven** [reply]
![2](https://user-images.githubusercontent.com/74394439/158718013-34fc1d2f-3acd-4492-8eb5-03aa0c826759.png)
ok, ya lo hice y la barra se extiende hasta el final de la paguina, tendre que aumentarle el tamaño al resto?. o hay una manera de definir el tamaño de la paguina o un zoom , gracias por tu tiempo

**Reply by LuisEsquivel27** [reply]
![image](https://user-images.githubusercontent.com/26752004/158718137-ea5274ac-f4de-473c-906e-401c830bda58.png)
Tendrías que ajustarlo o quitar esos width de tus clases por que veo que en .primerasec1 lo limitas a 500px, entonces igual deberías quitar esos width, como recomendación, veo en tu html al que tiene esa clase es un div, podrías usar la etiqueta section y así identificas más fácil tus secciones para que no te llenes de div y te puedas confundir, para el caso de los width que utilizas, cuando se requiera que sea responsivo, no es tan recomendable trabajar con pixeles, por lo mismo que se limita al número de pixeles sin importar las dimensiones del dispositivo, podrías usar % y eso calcula el % dependiendo del dispositivo

**Reply by skrillheaven** [reply]
ok lo checare, muchismas gracias!!!!!!!!!

**Reply by skrillheaven** [reply]
gracias por su ayuda, ahora ya se acomoda conforme a la pantalla, son los mejores, mucho exito!!!
![s](https://user-images.githubusercontent.com/74394439/158719936-ace25032-f667-4e49-adbf-9e33781230c4.jpg)

**Comment by skrillheaven** [comment]
creo que el problema puede cer el nav, hice lo que comentas y le deje el nav y se ve igual, sera las classes de boostrap que le agrege a ese apartado y por eso afecta a la pantalla?
![nav](https://user-images.githubusercontent.com/74394439/158717416-240727ef-a7d9-49c7-bf0c-05864688ac89.png)

**Comment by OscarPedroza** [comment]
Creo que el problema es que estas poniendo al body un máximo de 1000px y por eso lo toma como límite, intenta ponerle al body un valor de 100% para que te ocupe toda la pantalla.

![image](https://user-images.githubusercontent.com/83786890/158717981-91150039-8e18-47f6-af25-737ad52a5de3.png)
